src/scripts.js

```diff
@@ -24,34 +24,43 @@ let allCards = document.querySelector('.all-cards')
 let template = document.querySelector('.template').content;
 
 homeButton.addEventListener('click', cardButtonConditionals);
+homeButtonSmall.addEventListener('click', cardButtonConditionals);
 favButton.addEventListener('click', viewFavorites);
+favButtonSmall.addEventListener('click', viewFavorites);
 cardArea.addEventListener('click', cardButtonConditionals);
 
 window.onload = onStartup();
 
 function onStartup() {
-  // getData("http://localhost:3001/api/v1/users")
-  // getData("http://localhost:3001/api/v1/ingredients")
-  // getData("http://localhost:3001/api/v1/recipes")
-
-  let userId = (Math.floor(Math.random() * 49) + 1)
-  let newUser = users.find(user => {
-    return user.id === Number(userId);
-  });
-  user = new User(userId, newUser.name, newUser.pantry, ingredientsData)
-  pantry = new Pantry(newUser.pantry)
-  populateCards(cookbook.recipes);
-  greetUser();
+  const ingredientsResults = getIngredients()
+    .catch((error) => console.log(error))
+  const userResult = getUser()
+    .then((userObject) => {
+      user = userObject
+      new Pantry(user.pantry)
```

Comment:
What is this doing again?

Reply:
This is the only place we have access to the user object as an object so that we can give the new Pantry the user.pantry array. Outside of this .then the user is a promise.